.github/workflows/CI.yml

```diff
@@ -75,6 +75,8 @@ jobs:
           arch: ${{ matrix.arch }}
       - uses: julia-actions/julia-buildpkg@latest
       - uses: julia-actions/julia-runtest@latest
+        env:
+          JULIA_DEBUG: Main
```

Comment:
Main?

Reply:
That's the module where the tests are run.